### PR TITLE
CHF-273 Added readme file and Health Check capability implementation for Tyco-Door-Window-Sensor

### PR DESCRIPTION
**Pull request by pchomal**
Added readme file and Health Check capability implementation for "Tyco-Door-Window-Sensor".

### CODE REVIEW

**Comment by pchomal** [comment]
@jackchi @ShunmugaSundar 

Please review the changes and merge.

File: devicetypes/smartthings/tyco-door-window-sensor.src/tyco-door-window-sensor.groovy [hunk 1]
```diff
@@ -229,44 +230,42 @@ private Map getContactResult(value) {
...
+ * */
+def ping() {
+	return zigbee.readAttribute(0x001, 0x0020) // Read the Battery Level
+}
+
 def refresh()
 {
-	log.debug "Refreshing Temperature and Battery"
-	[
+	log.debug "refresh called"
```
Comment (jackchi) [inline devicetypes/smartthings/tyco-door-window-sensor.src/tyco-door-window-sensor.groovy]: Is refreshing battery and temperature not accurate?

**Comment by pchomal** [comment]
We thought "refresh called" to be correct as we were calling enrollResponse().
But "Refreshing Temperature and Battery" looks to be accurate. We have modified accordingly.

File: devicetypes/smartthings/tyco-door-window-sensor.src/tyco-door-window-sensor.groovy [hunk 2]
```diff
@@ -229,44 +230,42 @@ private Map getContactResult(value) {
 	]
 }
 
+/**
+ * PING is used by Device-Watch in attempt to reach the Device
+ * */
+def ping() {
+	return zigbee.readAttribute(0x001, 0x0020) // Read the Battery Level
```
Comment (jackchi) [inline devicetypes/smartthings/tyco-door-window-sensor.src/tyco-door-window-sensor.groovy]: @parijatdas Can we read the temperature here for consistency?

**Comment by ShunmugaSundar** [comment]
@jackchi 
Incorporated the changes. Please review.

**Comment by pchomal** [comment]
@jackchi @ShunmugaSundar 
Incorporated the changes. Please review.

**Comment by jackchi** [comment]
💯

**Pull request merged**